### DISCUSSION TITLE
Show tooltip with exact number when hovering over an abbreviated number

### DISCUSSION THREAD

**Discussion by kristjanmar** [plausible/analytics#1968] [Anything Else]
I like to keep track of my metrics in a spreadsheet. For this, I like to input the exact number.

It's useful for readability when Plausible shortens big numbers, for example changing 1923 to 1.9k.

However, I think the exact number should always show up in a tooltip/popover when hovering over the shortened number.

![CleanShot 2022-06-22 at 11 20 26](https://user-images.githubusercontent.com/22234692/174942971-e4e37ee0-693d-415f-996a-fc9d2f21a72f.png)

Is it possible to add this feature? The tooltip already shows for "Unique Visitors" and "Total Pageviews" at the top, but it would make my life a lot easier if it was also enabled for the other reports.

Like this:

![CleanShot 2022-06-22 at 11 26 12](https://user-images.githubusercontent.com/22234692/174943328-6a4eac20-bdf6-4cf8-8d4d-18e4da491f6f.png)

**Comment by metmarkosaric** [comment]
Thanks for the feedback @kristjanmar! We'll consider what can be done. In the meanwhile, you can get exact numbers using our [CSV export](https://plausible.io/docs/export-stats) or the [stats API](https://plausible.io/docs/stats-api).

**Comment by kristjanmar** [comment]
Thank you for adding this! A very nice UX improvement.
